src/ontology/mondo-ingest.Makefile

```diff
@@ -239,7 +242,7 @@ tmp/mondo.sssom.tsv:
 	wget http://purl.obolibrary.org/obo/mondo/mappings/mondo.sssom.tsv -O $@
 
 tmp/mondo.sssom.ttl: tmp/mondo.sssom.tsv
-	sssom convert $< -O rdf -o $@
+	sssom convert $< -O rdf -o $@tmp/mondo.ow
```

Comment:
st wrong here

Reply:
Glad you noticed that. Must have been a weird copy/paste error. Fixed.